pallets/vanchor/src/lib.rs

```diff
@@ -102,7 +103,7 @@ pub mod pallet {
 	#[pallet::config]
 	/// The module configuration trait.
 	pub trait Config<I: 'static = ()>:
-		frame_system::Config + pallet_linkable_tree::Config<I>
+		frame_system::Config + pallet_linkable_tree::Config<I> + pallet_key_storage::Config<I>
```

Comment:
I don't think we need to add it here if we're creating a `Config` parameter that implements the releated trait. Wdyt?